syntax = "proto3";

package jiji.rpc;

import "google/protobuf/empty.proto";

service LoggerService {

  rpc Log(LoggingRequest) returns (google.protobuf.Empty) {}

}

message LoggingRequest {
  string instance_id = 1;
  string log_level = 2;
  string message = 3;
}
